### DISCUSSION TITLE
Always problems when starting a new sensor - BT connection (?)

### DISCUSSION THREAD

**Discussion by bionictom** [Navid200/xDrip#60] [Dexcom]
Hi, whenever I'm trying to start a new sensor, it's not working properly, since I got my new phone in spring.

I started a new sensor yesterday afternoon, and ever since Xdrp is stuck on the "collecting Initial Readings" Box and the Step "Receiving Data from DexcomG5 Collector". With "No Data received".

I sometimes got a system popup, that said something like "Couldn't connect to Dexcom Bluetooth because of wrong password or Pin". Restarting and turning BT on/off didn't change anything.

The BT connection to the Rileylink works ok.

Here's a screenshot from the event log:

![grafik](https://user-images.githubusercontent.com/28786121/129147891-c933157d-849b-47b6-a40d-d63ddbc0a1b0.png)

**Comment by Navid200** [comment]
Do you mind showing screenshots of both classic and G5/G6 status pages.
If you like to blackout the serial number, please leave the first two characters visible.

What is the phone with xDrip on it?  What Android level is on it?

**Reply by bionictom** [reply]
It is a Xiaomi Poco F3 with Android 11

![grafik](https://user-images.githubusercontent.com/28786121/129188437-31dfd2b2-4847-4a1d-b7c4-1a948edaecb3.png)
![grafik](https://user-images.githubusercontent.com/28786121/129188452-a657730f-6494-4bfa-a12f-dc14a24b389a.png)

**Comment by Navid200** [comment]
The first thing I see is that the number of days has gone over 112.  This in itself is not an issue, if you are using non-native mode.
But, if you are using native algorithm, it will not work until you hard reset transmitter.

So, would you please show the top of the following page?
Settings -> G5/G6 Debug Settings

**Reply by bionictom** [reply]
I'm not using native mode
![grafik](https://user-images.githubusercontent.com/28786121/129189661-d7f9836e-e9c9-44d5-b8d1-d57110791108.png)

**Comment by Navid200** [comment]
Non-native.  Perfect.  No need to hard reset.
Would you please have a look here?
https://github.com/Navid200/xDrip/wiki/G5-Recommended-Settings

Are any of your settings different?

**Reply by bionictom** [reply]
All the settings are like in the link

**Comment by Navid200** [comment]
Do you see the log "prompting user to notice pairing ..." in the logs that you have posted?
It occurred at 5:50.
Did you get a pair request then?

**Reply by bionictom** [reply]
I'm not sure, either I didn't get one, or I didn't notice it. I was awake at 5:44 but might have already been sleeping again at 5:50

**Comment by Navid200** [comment]
Is there a watch involved?

**Reply by bionictom** [reply]
Yes, I do have a smartwatch, but it's only to display, i don't use it as receiver for the dexcom

**Comment by Navid200** [comment]
This is for troubleshooting?
If you disable the watch, removing it from the picture, do you get a pair request?

**Reply by bionictom** [reply]
I turned of the watch, restarted the phone.
Then I disabled and re-enabled bluetooth
Then i went to settings, switched to G4 as ahrdware source, and back to G5
After 2 minutes i got the dexcom pairing buttonand now Xdrip already got its first reading, so everything seems to work now.


So, next time it won't pair, should i turn off the watch until it works? I have done all the other steps, except for turning off the watch a few times since yesterday, but only now got the pairing request.

**Comment by Navid200** [comment]
Only jamorham can fix this as far as I can tell.  But, he doesn't have a facebook account.
I have a facebook account and am active helping people on facebook and see this issue all the time.  But, I have 0 experience and knowledge of wear.

This means I need to gather data before I can present it to him.  I don't want to waste his time before I have enough.
So, I have asked people to open an issue:
https://github.com/NightscoutFoundation/xDrip/issues/1789

Will you be so kind to post in that issue and state what you just experienced in a very short post including the transmitter, version of xDrip, your watch, your wear setting you disabled to get the pair request ...?

Thanks

**Reply by bionictom** [reply]
Thank you for your help, you've just made my live so much better by making my loop work again :)

**Reply by Navid200** [reply]
You are welcome.  Thanks for feedback.
